### PR TITLE
MountWeapons implemented buyable weapons; weapons can be upgraded and…

### PR DESCRIPTION
**Pull request by anhoppe**
… mounted to the player's tank

### CODE REVIEW

File: mountWeapon.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <script src="weapon.js"></script>
+        <script src="gun.js"></script>
+        <script src="playerStats.js"></script>
+        <title>Tanx2 - Return of the Killer Tanx</title>
+        <style>
```
Comment (Philipp-git) [inline mountWeapon.html]: A style.css would make the code cleaner.

File: spreadGun.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class SpreadGun extends Weapon
```
Comment (Philipp-git) [inline spreadGun.js:1]: Why not make all guns in a guns.js file?

File: mountWeapon.html (hunk 1 above)
Comment (anhoppe) [reply mountWeapon.html]: Ok, will do

File: spreadGun.js (hunk 2 above)
Comment (anhoppe) [reply spreadGun.js:1]: One file, one class, I don't like to put multiple classes into one file. The file structure should be reworked but I would not do this in the scope of this story

**Review by Philipp-git — APPROVED** [review]

**Pull request merged**